### ISSUE TITLE
This script's functionality overlaps with another

### ISSUE THREAD

**Issue by advocateddrummer** [pointwise/SplitConsAtIntersection#1]
@traviscarrigan, I came across this script by accident today whilst I was looking at [ConSplitAtIsect](https://github.com/pointwise/ConSplitAtIsect), which I have forked... I was looking at ways to fix an issue that has been bothering me and somehow I came across this code. I wonder, why are there two version of a script that seemingly do the exact same thing? It seems that this version is newer/nicer, but does not provide the same ability as the old version with respect to allowing the user to enter a tolerance manually. I think that's a valuable feature. Should these scripts be merged into a single version?

**Comment by RcktMan77** [comment]
@advocateddrummer I created the second, newer script before realizing there was an existing script that already provided similar functionality.  I also tend to prefer utility scripts that don't have GUIs attached as they more often than not interrupt my meshing workflow.  The script I created is a bit more limited, but it also gets the job done for 90% of the use cases that I encounter without prompting me with a GUI.  I can select two connectors, use the cmd+E keyboard shortcut, and the connectors are split.  Feel free to modify either script to suit your needs.
